API-LocateContractor/obj/Debug/net6.0/project.razor.json

```diff
@@ -12,7 +12,7 @@
   "ProjectWorkspaceState": {
```

Comment:
Seems your debug and bin folders a sneaking back in. Check your ignore and make sure your file tracking is clean

Reply:
I think I was able to take care of this.